awake.lua

```diff
@@ -387,6 +409,67 @@ function gridredraw()
   g:refresh()
 end
 
+function a.delta(n, d)
+  if n == 1 then
+    params:delta("pw", 0.2 * d)
+  elseif n == 2 then
+    -- cutoff is exponential; we use the controlspec
+    -- for adjustments to feel natural
+    local now = cs_CUT:unmap(params:get("cutoff"))
+    local delta = 0.002
+    params:set("cutoff", cs_CUT:map(now + delta * d))
+  elseif n == 3 then
+    params:delta("gain", 0.2 * d)
+  elseif n == 4 then
+    params:delta("release", 0.2 * d)
+  end
+end
+
+function arcredraw()
+  if not a then
+    return
+  end
+
+  -- Arc doesn't have built-in rotation support so we do
+  -- it ourselves.
+  local rot = 16 * (params:get("arc_rotation") - 1)
+  a:all(0)
+  a:led(1, rot + 1, 4)
+  a:led(2, rot + 1, 4)
+  a:led(3, rot + 1, 4)
+  a:led(4, rot + 1, 4)
```

Comment:
We show the top tick if no value is present. This lets the user see what Arc rotation is set.